### ISSUE TITLE
(Feature) Admin should be able to generate a new Confirmation Code

### ISSUE THREAD

**Issue by Janther** [poanetwork/poa-popa#83]
**continue the discussion here: #88**

- **What is it?** (leave one option)
    * New `(Feature)` request (i.e. some nice-to-have feature is missing)

- What feature are you talking about?**

The confirmation code is set up when a person registers a new address. If the system is going to randomly send postcards asking the user to re-confirm the address, I suppose you should be changing the confirmation code. Otherwise, a person could store the code and use it again and again.

- **What is proposed feature?**

A function for an admin to modify the existing confirmation code.

- **Additional information**

Also, a state for an Address that stipulates that it's waiting for reconfirmation would be nice.

**Comment by phahulin** [comment]
@pablofullana do we have this feature - reconfirming addresses - in our roadmap? At the present moment confirmation code is sent only once, after initial registration

**Comment by igorbarinov** [comment]
@phahulin yes, PoPA is a regular check. Thus, it's good to have a code unknown to the receiver for each attestation.

**Comment by pablofullana** [comment]
A few questions on this one:
1. Are we currently keeping track of the date in which user will be asked to re-confirm a given address? How is this defined?
1. Should an admin be in charge of changing the confirmation code or shall we generate them automatically?

**Comment by igorbarinov** [comment]
@pablofullana 
- it's defined by a user. We have an idea of `challenging validators` where any third party could ask validator to prove his identity and sponsoring that challenge. It's not our immediate need tho
- admin should not touch logic of the app. he's only deploying the app

**Comment by pablofullana** [comment]
Looks like enabling an address to be re-confirmed or re-registered will require some updates on the code. This shouldn't be that hard to implement, we could assign someone and start working on it.

My only concern is about the refactor you are working on @Janther and how this will impact it. Shall we plan on having this done after your updates or everything could be done in parallel?
